File: openmrs-client/src/main/java/org/openmrs/mobile/models/EncounterProvider.java
package org.openmrs.mobile.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class EncounterProvider extends Resource
{
    @SerializedName("provider")
    @Expose
    private Resource provider;

    @SerializedName("encounterRole")
    @Expose
    private Resource encounterRole;

    public Resource getProvider() {
        return provider;
    }

    public Resource getEncounterRole() {
        return encounterRole;
    }
}

File: openmrs-client/src/main/java/org/openmrs/mobile/models/EncounterProvider.kt
package org.openmrs.mobile.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class EncounterProvider : Resource() {
    @SerializedName("provider")
    @Expose
    var provider: Resource? = null
    @SerializedName("encounterRole")
    @Expose
    var encounterRole: Resource? = null

}